ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/flow_tracking_settings.schema.yml

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  flow_tracking_settings:
+    documentation_options:
+      table: flow-tracking-settings
```

Comment:
so this is not needed here as it will be generated automatically from the name. However the generated table must then be added to some place - probably for now in wan-preview. I can help you

Reply:
Didn't understand, updated table to wan-preview is this what you meant